WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsHubTimeRangeController.swift

```diff
@@ -0,0 +1,146 @@
+import Foundation
+import Yosemite
+
+struct TimeRange {
```

Comment:
I think this is worth being called `AnalyticsHubTimeRange` so other members are aware that this is specific to our module & need.

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8193/commits/5cb54d83dcf696381c435980ae77ad0822624bc3